features/section.feature

```diff
@@ -87,3 +87,7 @@ Feature: Page Sections
   Scenario: Get native property from section
     When I navigate to the home page
     Then I can obtain the native property of a section
+
+  Scenario: shadow-root section
+    When I navigate to the shadow root page
+    Then I can see elements from the section composed of Shadow Root
```

Comment:
"composed of shadow roots"

Reply:
I fixed, https://github.com/site-prism/site_prism/pull/159/commits/2f27655fe5b7e6bed2c6f34a4ee54b7bec301a54